phoenix-core/src/main/java/org/apache/phoenix/query/DefaultGuidePostsCacheFactory.java

```diff
@@ -29,11 +28,15 @@ public PhoenixStatsLoader getPhoenixStatsLoader(ConnectionQueryServices querySer
         Preconditions.checkNotNull(config);
 
         final boolean isStatsEnabled = config.getBoolean(STATS_COLLECTION_ENABLED, DEFAULT_STATS_COLLECTION_ENABLED);
-        if (queryServices == null || !isStatsEnabled) {
+        if (queryServices == null || ! isStatsEnabled) {
```

Comment:
extra space?

Reply:
I always add an extra space between ! and the variable for better readness. I saw the other places use this coding style too.